sys/include/net/gnrc/nettype.h

```diff
@@ -104,6 +105,21 @@ typedef enum {
      * @}
      */
 
+    /**
+     * @{
+     * @name Link layer
+     */
+#ifdef MODULE_GNRC_PPP
+    GNRC_NETTYPE_LCP,   /**< Protocol is PPP LCP */
+    GNRC_NETTYPE_IPCP,  /**< Protocol is PPP IPCP */
+    GNRC_NETTYPE_HDLC,  /**< Protocol is HDLC */
+    GNRC_NETTYPE_IPV4,  /**< Protocol is IPV4 encapsulated in HDLC frame */
```

Comment:
Considering that later on there might be IPv4 frames, too putting this into its own `#ifdef` and making `gnrc_ppp` dependent on a `gnrc_ipv4` pseudo-module might be the cleaner approach.